Sources/_TestingInternals/include/Discovery.h

```diff
@@ -96,7 +96,7 @@ SWT_EXTERN const void *_Nonnull const SWTTypeMetadataSectionBounds[2];
 ///
 /// - Returns: A pointer to an array of type metadata pointers. The caller is
 ///   responsible for freeing this memory with `free()` when done.
```

Comment:
", or nil if…"

Reply:
Well, no, it's the outer pointer that will be `nullptr` if we didn't end up finding any matching types.

Can tweak the comment.